specification/chaos/resource-manager/Microsoft.Chaos/preview/2024-03-22-preview/capabilities.json

```diff
@@ -0,0 +1,262 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Chaos Management",
+    "description": "Azure Chaos Resource Provider REST API",
+    "version": "2024-03-22-preview"
```

Comment:
>2024-03-22-preview [](http://example.com/codeflow?start=16&length=18)

Not really critical, but do you have to go with the version 2024-03-22-preview? API versions that don't use -01 as the day are unusual enough I've heard some opinions that we shouldn't use them to avoid causing confusion.

Reply:
For this version it's too late to change now. There are no guidance from ARM on what versioning best practices to use, so what we've been following is just to align the API version with the SDK release date we are aiming to hit. In March that would've been 22nd, the 4th Friday of the month. If we missed any documentation on this topic, can you please point us to the relevant docs? Thanks